src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final int radius;
+
+    public Circle(int radius) {
+        this.radius = radius;
+    }
```

Comment:
```suggestion
    public Circle(String color, int radius) {
        super(color);
        this.radius = radius;
    }
```
you can call parent class constructor